### PR TITLE
d_a_e_db_leaf OK

### PR DESCRIPTION
**Pull request by Yotona**

### CODE REVIEW

File: rel/d/a/e/d_a_e_db_leaf/d_a_e_db_leaf.cpp [hunk 1]
```diff
@@ -1,193 +1,94 @@
...
 
 /* 806A1EA0-806A1F24 0000E0 0084+00 2/1 0/0 0/0 .text daE_DB_LEAF_Execute__FP15e_db_leaf_class */
-#pragma push
-#pragma optimization_level 0
-#pragma optimizewithasm off
-static asm void daE_DB_LEAF_Execute(e_db_leaf_class* param_0) {
-    nofralloc
-#include "asm/rel/d/a/e/d_a_e_db_leaf/d_a_e_db_leaf/daE_DB_LEAF_Execute__FP15e_db_leaf_class.s"
+static int daE_DB_LEAF_Execute(e_db_leaf_class* leaf) {
+    PSMTXTrans(mDoMtx_stack_c::now, leaf->current.pos.x, leaf->current.pos.y, leaf->current.pos.z);
```
Comment (TakaRikka) [inline rel/d/a/e/d_a_e_db_leaf/d_a_e_db_leaf.cpp:15]: these calls can probably be simplified with the mDoMtx_stack_c inlines

File: rel/d/a/e/d_a_e_db_leaf/d_a_e_db_leaf.cpp [hunk 2]
```diff
@@ -1,193 +1,94 @@
...
+    mDoMtx_XrotM(mDoMtx_stack_c::now, leaf->shape_angle.x);
+    PSMTXCopy(mDoMtx_stack_c::now,(leaf->mpMorf->getModel()->mBaseTransformMtx));
+    leaf->mpMorf->modelCalc();
+    return 1;
 }
-#pragma pop
 
 /* 806A1F24-806A1F2C 000164 0008+00 1/0 0/0 0/0 .text daE_DB_LEAF_IsDelete__FP15e_db_leaf_class */
-static bool daE_DB_LEAF_IsDelete(e_db_leaf_class* param_0) {
+static bool daE_DB_LEAF_IsDelete(e_db_leaf_class* leaf) {
```
Comment (TakaRikka) [inline rel/d/a/e/d_a_e_db_leaf/d_a_e_db_leaf.cpp:24]: all actor methods should return int

File: rel/d/a/e/d_a_e_db_leaf/d_a_e_db_leaf.cpp [hunk 3]
```diff
@@ -1,193 +1,94 @@
...
-//
 
 /* 806A1E38-806A1EA0 000078 0068+00 1/0 0/0 0/0 .text daE_DB_LEAF_Draw__FP15e_db_leaf_class */
-#pragma push
-#pragma optimization_level 0
-#pragma optimizewithasm off
-static asm void daE_DB_LEAF_Draw(e_db_leaf_class* param_0) {
-    nofralloc
-#include "asm/rel/d/a/e/d_a_e_db_leaf/d_a_e_db_leaf/daE_DB_LEAF_Draw__FP15e_db_leaf_class.s"
+static int daE_DB_LEAF_Draw(e_db_leaf_class* leaf) {
```
Comment (TakaRikka) [inline rel/d/a/e/d_a_e_db_leaf/d_a_e_db_leaf.cpp:6]: preferably the param name for methods should be `i_this`

**Review by TakaRikka — COMMENTED** [review]
just a few things but looks good. also need to fix progress file conflict

**Comment by Yotona** [comment]
Fixed those issues. Not sure what happened to progress honestly, think maybe the tp script's pull-request calculation doesn't calculate rels

**Comment by hatal175** [comment]
Why doesn't github run the check on the newer cpmmits?

**Review by hatal175 — APPROVED** [review]

**Pull request merged**